lib/Demangling/Remangler.cpp

```diff
@@ -1379,6 +1379,16 @@ void Remangler::mangleIVarDestroyer(Node *node) {
   Buffer << "fE";
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+void Remangler::mangleImplDifferentiable(Node *node) {
+  Buffer << 'd';
+}
+
+// SWIFT_ENABLE_TENSORFLOW
```

Comment:
Shift ~4 lines below, missing ` END`.

Reply:
From above:
> That would be a better organization of these comments, but we haven't been following strict rules for these comments.

Optimizing/polishing these comments could be satisfying but they're not very important. We hope to upstream some of this code soon, too.

Reply:
Optimized comment in https://github.com/apple/swift/pull/26600.